File: python/2-conditional-struct/age-range/main.py
# Write a program to ask the user how old they are. Print whether he is a child, teenager, adult or elderly person.

age = int(input("Enter yout age: "))

if age >= 0 and age <= 12:
    print("\nYou are in the child age group.\n")
elif age >= 13 and age <= 17:
    print("\nYou are in the teenage age group.\n")
elif age >= 18 and age <= 64:
    print("\nYou are in the adult age group.\n")
elif age >= 65:
    print("\nYou are in the elderly age group.\n")
else:
    print("\nInvalid input!\n")

File: java/2-conditional-struct/age-range/src/Main.java
// Write a program to ask the user how old they are. Print whether he is a child, teenager, adult or elderly person.

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int age;

        System.out.print("Enter your age: ");
        age = sc.nextInt();

        if (age >= 0 && age <= 12) {
            System.out.println("\nYou are in the child age group.");
        }
        else if (age >= 13 && age <= 17) {
            System.out.println("\nYou are in the teenage age group.");
        }
        else if (age >= 18 && age <= 64) {
            System.out.println("\nYou are in the adult age group.");
        }
        else if (age >= 65) {
            System.out.println("\nYou are in the elderly age group.");
        }
        else {
            System.out.println("\nInvalid input!");
        }

    }
}